pkg/svcat/service-catalog/class.go

```diff
@@ -117,11 +125,55 @@ func (sdk *SDK) RetrieveClassByPlan(plan *v1beta1.ClusterServicePlan,
 	return class, nil
 }
 
-// CreateClass returns new created class
-func (sdk *SDK) CreateClass(class *v1beta1.ClusterServiceClass) (*v1beta1.ClusterServiceClass, error) {
-	created, err := sdk.ServiceCatalog().ClusterServiceClasses().Create(class)
+// CreateClassFrom returns new created class
+func (sdk *SDK) CreateClassFrom(opts CreateClassFromOptions) (Class, error) {
+	fromClass, err := sdk.RetrieveClassByName(opts.From)
```

Comment:
Let's say that someone want to copy an existing namespaced class. How would they go about doing this? For now it may be safe to assume that they can only copy a namespaced class from the same namespace they are creating the new class in, in which case you will want to take into account `opts.Namespace` when retrieving the class here.